Changed messages after importing unlinked files to passive.

Racheal7 opened the pull request:
Fixes [koppor#548](https://github.com/koppor/jabref/issues/548). This issue is about the messages displayed in the dialog after importing a file that may cause the user to think that the process is still in progress. 

![issue 548](https://user-images.githubusercontent.com/45301647/197378931-d344f4bb-65b0-49ea-929f-85418ee4b57d.png)

I have changed the importing messages to passive, which would be more intuitive for users. 

![update message](https://user-images.githubusercontent.com/45301647/197378992-1d5a5067-5cd9-495b-97f6-f06792f9e480.png)

When no metadata is found, the message was changed from "No metadata found. Creating empty entry with file link" to "No metadata found. Created an empty entry with file link"

Same for importing the BibTex entries. When an entry is successfully imported, the message was changed from "importing bib entry" to "Successfully imported bib entry". If no metadata is found, the message was changed from "No BibTeX data found. Creating empty entry with file link" to "No BibTeX data found. Created an empty entry with file link".

![update message for bibtext entry](https://user-images.githubusercontent.com/45301647/197379566-3e723be5-250a-4f1b-8c14-e341d3daa082.png)

In addition, I think the icon for successfully importing the BibTex file should be the same as importing a PDF file. It might be a bug that needs to be fixed. 

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

Commit:
Changed messages after importing unlinked files to passive.

Gpax971 commented:
These are more past tense than passive. Passive would be "Bib entry was successfully imported" rather than "Successfully imported bib entry"

Racheal7 commented:
@Gpax971 
Ah yeah, you are right. Will change it now.

Racheal7 commented:
@Gpax971 After reviewing the issue, I am a little confused because Oliver Kopp's proposal in the issue is "The message should contain "created new entry" or "merged into existing entry". So I guess he meant to change the message to past tense rather than passive?

Gpax971 commented:
You may wish to ask for clarification on the issue page then

ThiloteE committed:
Replace GitHub action "Greeting" with "GreetingFirstTimeCodeContribution" (#9283)

* Delete non-working action

Deletes greetings.yml

* Add new funktioning action

adds GreetingFirstTimeCodeContribution.yml

Commit:
Changed icon for successfully import bib entry to "check" icon.

Racheal7 commented:
The icon for importing bibtex has been changed to "check".

![changed icon](https://user-images.githubusercontent.com/45301647/197388793-73ae1300-56a4-4ff0-ab72-7094fe7449bb.png)

Commit:
Changed messages after importing to past passive tense.
Modified CHANGLOG.md to reflect the change.

dependabot[bot] committed:
Bump hmarr/auto-approve-action from 3.0.0 to 3.1.0 (#9295)

Bumps [hmarr/auto-approve-action](https://github.com/hmarr/auto-approve-action) from 3.0.0 to 3.1.0.
- [Release notes](https://github.com/hmarr/auto-approve-action/releases)
- [Commits](https://github.com/hmarr/auto-approve-action/compare/v3.0.0...v3.1.0)

---
updated-dependencies:
- dependency-name: hmarr/auto-approve-action
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump mockito-core from 4.8.0 to 4.8.1 (#9300)

Bumps [mockito-core](https://github.com/mockito/mockito) from 4.8.0 to 4.8.1.
- [Release notes](https://github.com/mockito/mockito/releases)
- [Commits](https://github.com/mockito/mockito/compare/v4.8.0...v4.8.1)

---
updated-dependencies:
- dependency-name: org.mockito:mockito-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump pascalgn/automerge-action from 0.15.3 to 0.15.5 (#9296)

Bumps [pascalgn/automerge-action](https://github.com/pascalgn/automerge-action) from 0.15.3 to 0.15.5.
- [Release notes](https://github.com/pascalgn/automerge-action/releases)
- [Commits](https://github.com/pascalgn/automerge-action/compare/v0.15.3...v0.15.5)

---
updated-dependencies:
- dependency-name: pascalgn/automerge-action
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump lucene-core from 9.4.0 to 9.4.1 (#9301)

* Bump lucene-core from 9.4.0 to 9.4.1

Bumps lucene-core from 9.4.0 to 9.4.1.

---
updated-dependencies:
- dependency-name: org.apache.lucene:lucene-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

* update all lucence deps
works

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Siedlerchr <siedlerkiller@gmail.com>

koppor committed:
Have fetcher tests running only if fetchers are touched

Gpax971 committed:
Download from URL now automatically fills with URL in clipboard (#9282)

Autofills URL in clipboard when downloading from URL
Fixes koppor#535

Racheal7 committed:
Add file path in the Delete dialog text. (#9284)

Instead of absolute file path, only the file name is showed in the dialog title. The file's absolute path is added in the dialog text.

Fixes https://github.com/koppor/jabref/issues/534.

Racheal7 committed:
Merge branch 'main' into fix-for-issue-548

Siedlerchr committed:
Update CHANGELOG.md

koppor commented:
This PR can be closed - there is an upstream PR https://github.com/JabRef/jabref/pull/9308

Congratz that the upstream PR was merged!
